### DISCUSSION TITLE
Dual remote PTT support

### DISCUSSION THREAD

**Discussion by tiagofreire-pt** [egzumer/uv-k5-firmware-custom#184] [Ideas]
I'm searching for support, if it is possible to code the firmware to allow dual remote PTT switches, such the ones supported by the Baofeng UV-82: 
![image](https://github.com/egzumer/uv-k5-firmware-custom/assets/41837236/c082ab75-80de-4af9-b3d1-7a2496ca9d1b)

In: https://www.aliexpress.com/item/4001119317154.html

**Comment by ErikS-web** [comment]
What's inside that REMOTE?

I have a great&cheap external microphone (**Baofeng K-780MIC**) with an extra PTT (round button) on top. It does nothing on the UV-K5, only PTT.

The microphone of my Kenwood transciever (**SMC-33**) has 3 extra buttons in addition to the PTT. But they do nothing on the UV-K5, only PTT.

**Comment by ErikS-web** [comment]
Look like this...

![tea-like-u94-dual-ptt](https://github.com/egzumer/uv-k5-firmware-custom/assets/148579604/6b602fc4-463a-4847-a1fd-5cb12f5c3d4e)

**Comment by ErikS-web** [comment]
More info here ... https://www.miklor.com/COM/UV_Technical.php

The interface is NOT the standard Kenwood mic.

**Comment by egzumer** [comment]
No way. There should be another MCU pin connected somewhere to a radio socket internally, and I know nothing about it. Nothing indicates that.

![image](https://github.com/egzumer/uv-k5-firmware-custom/assets/14902414/08d61665-7f07-4f8e-aa09-74ffd0240818)

**Reply by ErikS-web** [reply]
So check if an MCU is not used and design a hardware modification
